.gitignore

```diff
@@ -3,3 +3,5 @@
 .vscode
 *.idea
 *node_modules
+
+tutorials/mobile/flutter-graphql/app-final/.packages
```

Comment:
Can we add this to the app-final .gitignore instead of being on the parent .gitignore?

Reply:
Done